docs/template-helpRequest.md

```diff
@@ -0,0 +1,41 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+**Step in the `settingUp.md` Guide / Failing Tests**
+
+*If you are setting up the project:*
+> Tell us the step in the setting up guide at which you are stuck
+
+*If you are resolving test failures:*
+> Tell us which tests are failing
+
+**Error Messages Shown...**
+
+*...in the TestNG tab, if you are resolving test failures:*
+> e.g. Failed tests:
+> parallel-ui-tests -> teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUITests
+
+*...in the Console, for other cases:*
+> e.g. Error Message:
+>
+> Re-trying removeAndRestoreDataBundle -
+> java.net.ConnectException: Connection refused
+
+**How You Tried Solving the Problem**
+
+
+**Operating System**
+
+
+**JDK Version**
+
+
+**Eclipse Version**
+
+
+**Firefox / Chrome Version (where applicable)**
```

Comment:
Everything from "Operating System" to "Firefox/Chrome Version" should be on top (right after Issue Title); it's good to get the system configurations early.
Also, (again) although quite obvious, you may want to add an explanation to the section "How you tried solving the problem".